gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/UserRolesResource.java

```diff
@@ -79,6 +76,7 @@ public void list(
             @PathParam("environmentId") String environmentId,
             @PathParam("domain") String domain,
             @PathParam("user") String user,
+            @QueryParam(value = "dynamic") @DefaultValue(value = "false") boolean dynamic,
```

Comment:
this new query parameter isn't used in the UI so the roles assigned by the RoleMapper aren't displayed.
IMHO we have to make two calls from the UI to get all roles and disable deletion for dynamic one.
Or maybe we can remove this parameter and always return the both type of roles.
In any cases, we have to return a 400 erro on role deletion if the role is part of the dynamic ones.

wdyt?

Reply:
I may go on the first solution and making 2 calls.
Since the separation is made, I'll keep it simple and display them as well